### ISSUE TITLE
Acrylic Effect Radius Changer doesn't change anything

### ISSUE THREAD

**Issue by Gurrrra** [ramensoftware/windhawk-mods#490]
@Dulappy
This mod doesn't change my blur radius no matter what I do.
The mod says that "you must enable Windhawk to inject into system processes in its advanced settings", but going into the advanced settings it's not clear what exactly that is. I've tried setting "Load mods in critical system processes" to "If targeted explicitly or implicitly (not recommended)" (even though I'm not exactly sure what that means), but it doesn't make the mod do anything.
Also tried to remove the "<critical_system_processes>" under "Process exclusion list" but no difference there either.
Apart from those two I cannot find anything under advanced that is related to system processes.

I was told here to use error logging, but whatever the instruction says I should enable to get those error logs isn't how it actually looks in the program. Tried to enable some kind of error logging though, but I have no idea where to find those logs..

And yes I'm running Win10 with a freshly installed Windhawk (not portable).

**Comment by Dulappy** [comment]
Everything you did was correct, I haven't updated the instructions to comply with the latest version of Windhawk.

Do you happen to be running DWMBlurGlass?

**Comment by Gurrrra** [comment]
Yes I am actually, but wanted to try the Windhawk way since DWMBlurGlass was a bit buggy. Did disable and uninstall it beforehand though, but maybe it did interfere anyways?

**Comment by Dulappy** [comment]
Ah, if DWMBlurGlass wasn't active then it didn't interfere. It's probably an issue with symbols. I'll have to update those instructions eventually. I'll let you know when I do.

**Comment by Kradek200** [comment]
Hi! I have the same issue, are you still planning to fix it in the future?

**Comment by technovore007** [comment]
It's been a year now still no update?
